src/utils/Request.ts

```diff
@@ -0,0 +1,72 @@
+enum METHODS {
+    GET = "GET",
+    PUT = "PUT",
+    POST = "POST",
+    DELETE = "DELETE"
+}
+
+function queryStringify(data: Record<string, string>) {
+	const entries = Object.entries(data).map(
+		([key, val]) => (`${key}=${val}`)
+	);
+	return `?${entries.join("&")}`;
```

Comment:
не critical, но можно записать красивее
 ```suggestion
	return "?" + Object
 .entries(data)
 .map(([key, value]) => `${key}=${value}`)
 .join("&")
```